accelerator/config.h

```diff
@@ -53,6 +53,8 @@ extern "C" {
   "AMRWQP9BUMJALJHBXUCHOD9HFFD9LGTGEAWMJWWXSDVOF9PI9YGJAPBQLQUOMNYEQCZPGCTHGV" \
   "NNAPGHA"
 #define MAM_FILE_PREFIX "/tmp/mam_bin_XXXXXX"
+#define BUFFER_LIST_NAME "txn_buff_list"
+#define DONE_LIST_NAME "done_txn_buff_list"
```

Comment:
Why do we have two list names?

Reply:
There are two list in the current implementation.
1. BUFFER_LIST: This list buffers the unsent transactions
2. DONE_LIST: This list buffers the sent transactions which are waiting for clients fetch transaction information with UUID.

Reply:
You should explain within relevant source.

Reply:
We have relevant descriptions at line 95 and line 96 in `config.h`

Reply:
I have relevant descriptions at line 95 and line 96 in `config.h`